helix-core/src/test/java/org/apache/helix/integration/manager/ClusterManager.java

```diff
@@ -74,7 +81,8 @@ public void syncStart() {
 
     _watcher = new Thread(this);
     _watcher.setName(String
-        .format("ClusterManager_Watcher_%s_%s_%s", _clusterName, _instanceName, _type.name()));
+        .format("ClusterManager_Watcher_%s_%s_%s_%d", _clusterName, _instanceName, _type.name(), _uid));
+    LOG.info("ClusterManager_watcher_{}_{}_{}_{} started, stacktrace {}", _clusterName, _instanceName, _type.name(), _uid, Thread.currentThread().getStackTrace());
```

Comment:
It seems this log is for your debugging. Do we want to remove it then?

Reply:
No, we don't. We specifically need this.

Reply:
This is testing code. Make it info won't pollute log. Will change to debug anyway.